File: Kotlin/URI_1036/Bhaskara_1036.kt

fun Bhaskara(A:Double,B:Double,C:Double){
    val delta = Math.pow(B,2.0) -4*A*C
    if (A == 0.0 || delta < 0.0 )  {println("Impossivel calcular")}
    else{
        val r_delta = Math.pow(delta,0.5)
        val r1 = (-B + r_delta) / (2.0*A)
        val r2 = (-B - r_delta) / (2.0*A)
        println("R1 = ${r1}")
        println("R2 = ${r2}")
    }
}

fun main(){
    val(A,B,C)= readLine()!!.split(" ").map(String::toDouble)
    Bhaskara(A,B,C)
}

File: Python/URI_1036/Bhaskara_1036.py
def Bhaskara(A,B,C):
    delta = B**2 - 4*A*C
    if delta < 0 or A == 0 :
        print("Impossivel Calcular")
    else:
        print("R1 = {}".format((-B + delta**0.5)/(2*A)))
        print("R2 = {}".format((-B - delta**0.5)/(2*A)))

def main():
    A,B,C = input().split(" ")
    Bhaskara(float(A),float(B),float(C))
main()